space.go

```diff
@@ -15,6 +15,7 @@ type spaces struct {
 
 type space struct {
 	Id_         string `yaml:"id"`
```

Comment:
With the addition of uuid, can't we do away with Id now?

Reply:
answered below